modmail/bot.py

```diff
@@ -25,7 +26,7 @@ def __init__(self, **kwargs):
         super().__init__(command_prefix=self.get_prefix, **kwargs)
         self.http_session = ClientSession()
 
-    async def get_prefix(self, message: discord.Message = None) -> str:
+    async def get_prefix(self, message: discord.Message = None) -> t.List[str]:
```

Comment:
TBH this should be in a separate PR.

Reply:
Not sure if it's required for lints to pass though. If it is, someone ought push this to main and merge that in so lints pass.

Reply:
> TBH this should be in a separate PR.

I thought it won't cause much different keeping this small change, I think lets keep it in this PR for the moment and I can remember that for future.

> Not sure if it's required for lints to pass though. If it is, someone ought push this to main and merge that in so lints pass.

No, lint won't fail on this as the linters don't check whether the annotations are correct or not.

Reply:
In the future, this kind of edit should be in a separate pr.

Even more so, given that this is a site pr and not a bot pr.

Its okay to merge in this case.